index.html

```diff
@@ -9035,6 +9041,74 @@ <h2>Screen capture</h2>
  and a <a>command</a> <a>Take Element Screenshot</a>
  for doing the same with the visible region of an <a>element</a>’s
  <a>bounding rectangle</a> after it has been <a>scrolled into view</a>.
+ It also provides a <a>Take Fullpage Screenshot</a> <a>command</a>
+ to optionally capture the <a>fullpage region</a>.
+
+<p>An <a>element</a>’s <dfn data-lt="fullpage region">fullpage region</dfn>
+ is the rendered rectangle that encompasses the maximal scrollable width
+ and height,
+ it can be calculated this way:
+<ol>
+ <li><p>Let <var>x</var> be 0
+
+ <li><p>Let <var>y</var> be 0
+
+ <li><p>Let <var>width</var> be the value
+  returned by
+  <p><a>max</a>(document.body.scrollWidth, document.documentElement.scrollWidth,
```

Comment:
Are there really no defined relations that enforce an ordering between any of these properties?

Reply:
I am not sure what you mean regarding ordering of the properties. I took this "heuristics" from https://javascript.info/size-and-scroll-window. In reality, ChromeDriver implementation will probably utilize CDP https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-getLayoutMetrics and use the contentSize width and height directly.